src/SASjs.ts

```diff
@@ -573,22 +573,16 @@ export default class SASjs {
    * @param params - request URL parameters.
    * @param overrideSasjsConfig - object to override existing config (optional)
    */
-  public uploadFile(
+  public async uploadFile(
     sasJob: string,
     files: UploadFile[],
     params: any,
     overrideSasjsConfig?: any
   ) {
-    const fileUploader = overrideSasjsConfig
-      ? new FileUploader(
-          { ...this.sasjsConfig, ...overrideSasjsConfig },
-          this.jobsPath,
-          this.requestClient!
-        )
-      : this.fileUploader ||
-        new FileUploader(this.sasjsConfig, this.jobsPath, this.requestClient!)
```

Comment:
It's not helpful to create new `FileUploader` instances. We already have one ( instantiated through `SASjs` constructor) and will always be present.